cg/cli/workflow/mip_dna/base.py

```diff
@@ -272,33 +261,124 @@ def run(
         raise click.Abort
 
 
-@mip_dna.command()
+@mip_dna.command("resolve-compression")
+@ARGUMENT_CASE_ID
 @OPTION_DRY
 @click.pass_context
-def start(context: click.Context, dry_run: bool = False):
-    """Start all cases that are ready for analysis"""
+def resolve_compression(context: click.Context, case_id: str, dry_run: bool):
+    """Handles cases where decompression is needed before starting analysis"""
     dna_api = context.obj["dna_api"]
-    exit_code = EXIT_SUCCESS
+    prepare_fastq_api = context.obj["prepare_fastq_api"]
+    case_obj = dna_api.db.family(case_id)
```

Comment:
Hi, I think this is very implicit. Could you expose what `dna_api.db` is? Perhaps with something like
```suggestion
    status_db: Store = dna_api.db
    case_obj: models.Family = status_db.family(case_id)
```